web/js/playground.js

```diff
@@ -11,6 +11,8 @@ import { storeManifestLink, getStoredManifestLinks } from './manifestStorage.js'
 
 const RECENTLY_USED_KEY = 'recentlyUsedTools';
 
+console.log("script is loading...");
```

Comment:
These logs in the console are helpful even when working on other things, to make sure that functions are still running properly. Maybe they should be commented out in a more final version of the webpage, though I wonder if I should add some of them to whatever functionality I may be working on as well for now.